### DISCUSSION TITLE
Indeterminate progress

### DISCUSSION THREAD

**Discussion by bayang** [saadeghi/daisyui#788] [Q&A] [answered]
Hi !

Thanks for your project, excuse me but if I use an indeterminate progress I can't see the animation, is it normal or what am I doing wrong ?

```
<progress
        class="progress progress-success mt-5"
      />
```

I'm using 2.14.3

**Comment by saadeghi** [comment]
If you can see the animation [here](https://play.tailwindcss.com/TQzgB39vLX),  please share a repo so I can reproduce your code and see what's the issue,

**Reply by bayang** [reply]
Thanks for having a look, actually I searched a bit more and it happens that the progress works on vivaldi (so chrome) but not on firefox.

I am able to reproduce with my code and also with your sample playground.

And I'm developing in firefox so I thought it was not working at all, maybe that can give you a hint ?

**Comment by saadeghi** [comment, marked as answer]
Okay, Looks like indeterminate progress can not have `:after` pseudo element in Firefox 🤔  
I think I should replace it with background gradient to make it work on Firefox too.  
I opened an issue here: https://github.com/saadeghi/daisyui/issues/794
